kalkulator/src/main/java/no/nav/folketrygdloven/kalkulator/guitjenester/fakta/VurderMottarYtelseDtoTjeneste.java

```diff
@@ -94,7 +95,8 @@ private void lagFrilansDel(BeregningsgrunnlagDto beregningsgrunnlag,
                                Optional<FaktaAktørDto> faktaAktør) {
         vurderMottarYtelseDto.setErFrilans(VurderMottarYtelseTjeneste.erFrilanser(beregningsgrunnlag));
         beregningsgrunnlag.getBeregningsgrunnlagPerioder().get(0).getBeregningsgrunnlagPrStatusOgAndelList().stream()
-                .filter(andel -> andel.getAktivitetStatus().erFrilanser()).findFirst()
+                .filter(andel -> andel.getAktivitetStatus().erFrilanser()
+		                && (AndelKilde.PROSESS_START.equals(andel.getKilde()) || AndelKilde.SAKSBEHANDLER_KOFAKBER.equals(andel.getKilde()))).findFirst()
```

Comment:
Litt usikker på om det er riktig å ta med dei med kilde SAKSBEHANDLER_KOFAKBER her., men trur ikkje det speler så stor rolle for oss akkurat no.

Reply:
Ok, jeg tok de med siden jeg så at AndelerForFaktaOmBeregningTjeneste(Linje 39) tok de med, men det er kanskje unødvendig der også?